src/trackedit/internal/trackeditactionscontroller.cpp

```diff
@@ -753,6 +773,8 @@ bool TrackeditActionsController::canReceiveAction(const ActionCode& actionCode)
 {
     if (globalContext()->currentProject() == nullptr) {
         return false;
+    } else if (recordController()->isRecording() && isDisallowedDuringRecording(actionCode)) {
```

Comment:
just a suggestion, maybe we should add `isRecording` to the global context?  
It might just be a little more convenient, fewer dependencies, for example less mocking and configuring when trying to write a unit test

Reply:
Makes sense.